src/codegen.cpp

```diff
@@ -357,6 +357,7 @@ static Function *jlboundserrorv_func;
 static Function *jlcheckassign_func;
 static Function *jldeclareconst_func;
 static Function *jlgetbindingorerror_func;
+static Function *jlboundp_func;
```

Comment:
that explains where the name came from, but doesn't help anyone reading the code who doesn't already know that figure out at a glance what it does